drivers/pinctrl/pinctrl_npcx.c

```diff
@@ -140,3 +170,5 @@ SYS_INIT(npcx_pinctrl_init, PRE_KERNEL_1, CONFIG_KERNEL_INIT_PRIORITY_DEFAULT);
 
 BUILD_ASSERT(sizeof(pinctrl_soc_pin_t) == 4, "Unexpected pinctrl_soc_pin_t size");
 BUILD_ASSERT(sizeof(struct npcx_periph) == 2, "Unexpected npcx_periph size");
+BUILD_ASSERT(DT_NUM_INST_STATUS_OKAY(DT_DRV_COMPAT) == 1,
+	"only one 'nuvoton,npcx-pinctrl' compatible node may be present");
```

Comment:
pinctrl is a singleton by definition, so this is redundant

Reply:
Done.